build-logic/testing-convention/src/main/kotlin/convention.gradle-testing.gradle.kts

```diff
@@ -34,12 +34,40 @@ val gradleTestTask = tasks.register<Test>("gradleTest") {
 
     useJUnitPlatform()
 
-    @Suppress("MagicNumber")
-    maxParallelForks = 8
+    /**
+     * timings provided only as relative to each other
+     * gradleTest:
+     *  1: 2m14s
+     *  16: 3m16s
```

Comment:
I'd like to see also how to optimize and objective constraints - CPU, memory?
Exact values can be optimal only to one machine but not for others.
Comparing only them without explaining constraints makes exact values less reusable.

Reply:
I tried to explain my decision better in comment, take a look